src/service/download.service.ts

```diff
@@ -0,0 +1,28 @@
+import { readFileSync, existsSync, mkdirSync, writeFileSync } from 'fs';
+import { resolve } from 'path';
+import 'isomorphic-fetch';
+import * as e6p from 'es6-promise';
+(e6p as any).polyfill();
```

Comment:
why do you use this?

Reply:
I'm using es6, and with using Promise with fetch()  function. In that way, I found Promise is [not supported](https://caniuse.com/#search=promise) in all IE versions, by that reason is why I'm using polyfill. The short answer is to be able to run test in those not supported web browsers.